### ISSUE TITLE
installation on windows(preview) need update.

### ISSUE THREAD

**Issue by zw963** [crystal-lang/crystal-website#402]
## Bug Report

https://crystal-lang.org/install/on_windows/

We need install git before following installation steps.

```sh
scoop install git
scoop bucket add crystal-preview https://github.com/neatorobito/scoop-crystal
scoop install crystal
```
for completeness, i consider we should add `scoop install vs_2022_cpp_build_tools` too.

Tried on a new install win10.

**Comment by zw963** [comment]
Probably new version scoop dependency.

when run `scoop bucket add crystal-preview https://github.com/neatorobito/scoop-crystal`,  error message tips need `scoop install git`.

**Comment by beta-ziliani** [comment]
fixed by #403
